quantities/README.md

```diff
@@ -0,0 +1,69 @@
+# Quantity components
+
+## Quantity design specs
+
+You can find the design specs on [decathlon.design](https://www.decathlon.design/).
+
+## Usage
+
+If you want to use components of this module in your android mobile application, you should
+first add the Gradle dependency in your Gradle file:
+
+```kotlin
+implementation("com.decathlon.vitamin.compose:quantities:<versions>")
+```
+
+### Default
+
+```kotlin
+object VitaminQuantities {
+    @Composable
+    fun Default(
```

Comment:
Like described in our conventions (https://github.com/Decathlon/vitamin-compose/blob/main/CONVENTIONS.md#composable-variants), if you have only one variant, it should be named `Primary`.